source/includes/_endpoint-dataset.md

```diff
@@ -686,96 +686,66 @@ Refer to the options described on the table above for the `csv` format to change
 
 
 ##### Match
-`GET /datasets/match{?datasets={id}&datasets={id}[&datasets={id}]+`
+```http
```

Comment:
This needs a bit more information. The controller sets a Location header somewhere, and the endpoint can return 201 or 202 sometimes. It is not specified when should one expect such response and what to do on each case